content/applications/productivity/calendar/outlook.rst

```diff
@@ -2,37 +2,65 @@
 Outlook Calendar synchronization
 ================================
 
-Synchronizing a user's Outlook Calendar with Odoo is useful for keeping track of their tasks and
+Synchronizing a user's Outlook Calendar with Odoo is useful for keeping track of tasks and
 appointments across all related applications.
 
 .. seealso::
    - :doc:`/applications/general/users/azure`
    - :doc:`/applications/general/email_communication/azure_oauth`
 
-Register the application with Microsoft Azure
-=============================================
+Setup in Microsoft Azure
+========================
 
 To sync the Outlook Calendar with Odoo's Calendar, a Microsoft Azure account is needed. Creating an
 account is free for users who have never tried or paid for Azure. For more information, `click here
 <https://azure.microsoft.com/en-us/free/?WT.mc_id=A261C142F>`_.
 
 Refer to `Microsoft's documentation <https://docs.microsoft.com/en-us/azure/active-directory/
-develop/quickstart-create-new-tenant>`_ on how to set up an Microsoft Entra ID (formally called
-*Microsoft Azure Active Directory (Azure AD)*), which is a representation of an organization to
-manage and register apps.
+develop/quickstart-create-new-tenant>`_ on how to set up a Microsoft Entra ID (formally called
+*Microsoft Azure Active Directory (Azure AD)*). This is a API console to manage and register
+Microsoft applications.
 
-Then, `Register an Application <https://docs.microsoft.com/en-us/azure/active-directory/develop/
-quickstart-register-app>`_, choosing the appropriate :guilabel:`Supported account type`. Users who
-wish to connect their Outlook calendar to Odoo should select the :guilabel:`Accounts in any
-organizational directory (Any Microsoft Entra ID directory - Multitenant) and personal Microsoft
-accounts (e.g. Skype, Xbox)` option for :guilabel:`Supported account types`.
+Existing Microsoft Entra ID users should log in at the `Microsoft Azure developer portal
+<https://portal.azure.com/#home>`_. Next, select :guilabel:`View` under the section labeled *Manage
+Microsoft Entra ID*.
```

Comment:
```suggestion
<https://portal.azure.com/#home>`_. Next, select :guilabel:`View` under the section labeled :guilabel:`Manage
Microsoft Entra ID`.
```
adding a GUI because it's the name of a section that appears on the UI